interface/new/new_comprehensive.php

```diff
@@ -40,7 +40,7 @@ function getLayoutRes()
     global $SHORT_FORM;
     return sqlStatement("SELECT * FROM layout_options " .
     "WHERE form_id = 'DEM' AND uor > 0 AND field_id != '' " .
-    ($SHORT_FORM ? "AND ( uor > 1 OR edit_options LIKE '%N%' ) " : "") .
+    ($SHORT_FORM ? "AND ( uor > 1 OR edit_options LIKE BINARY '%N%' ) " : "") .
```

Comment:
Checking case is a good idea but since you found null values it should be like this:
($SHORT_FORM ? "AND ( uor > 1 OR edit_options IS NOT NULL AND edit_options LIKE BINARY '%N%' ) " : "") .

Reply:
This should be okay the way it is unless the legacy layout was changed and the N modifier was changed to LC n. In which case, it should be fixed in layout and look for others, otherwise all the sql's for modifiers would need to be changed to test case e.t.c. Also, don't think null should effect this query but correct me if i'm wrong. Am not including in my isOption fix.